bodhi/server/scripts/compat.py

```diff
@@ -94,6 +94,18 @@ def check_policies(config_uri):
     _trigger_task(config_uri, "check_policies_task")
 
 
+@click.command()
+@config_uri_argument
+@click.version_option(message='%(version)s')
+def check_signed_builds(config_uri):
```

Comment:
We had this as a way to keep backward compatibility with older script run by cron jobs. Since this tasks is new I am not 100% sure we need that compatibility layer